presto-orc/src/main/java/com/facebook/presto/orc/metadata/Footer.java

```diff
@@ -62,6 +73,11 @@ public int getRowsInRowGroup()
         return fileStats;
     }
 
+    public Map<String, Slice> getUserMetadata()
+    {
+        return userMetadata;
```

Comment:
Not nice.
Maybe we should use sth else instead of slice after all?
I started with `String` but I changed that to `Slice`. AFAIR I did that because it was not in-line with approach in `OrcRecordWriter` - which uses ByteBuffer for metadata. And also a bit less general.

Maybe we should come back to `String`s. Or read-only `ByteBuffer`s?

What do you think?

Changing to copy for now.